meson.build

```diff
@@ -135,11 +143,17 @@ if get_option('ipc').enabled()
         c_args: ipc_lib_c_args,
         gnu_symbol_visibility: 'hidden',
     )
+    pkg.generate(
+        nanoarrow_ipc_lib,
+        filebase: 'nanoarrow-ipc',
```

Comment:
I am possibly overlooking or misunderstanding the current CMake configuration, but I couldn't find anywhere that we were producing pkgconfig files. 

As such, its worth double checking if we want to use hyphens or underscores for the pkgconfig file name. I am using hyphens here as that is what arrow does (ex; `arrow-compute.pc`)

Reply:
This comment might have been dropped by subsequent changes, so just want to make sure you see it @paleolimbot in case you disagree with the chosen naming convention